add-on/src/lib/ipfs-companion.js

```diff
@@ -787,6 +793,10 @@ module.exports = async function init () {
       return uploadResultHandler
     },
 
+    get openWebUiAtDirectory () {
+      return openWebUiAtDirectory
+    },
```

Comment:
Ugh, another realization: we can't open webui if embedded js-ipfs is used in non-Brave browser.
We need to check if `state.nodeType === 'embedded'` and open resource at public gateway instead.

Reply:
Perhaps we could make this even more useful:
What if we add toggle "Open imported files in WebUI"?
Enabled by default, but when disabled it would open at Gateway (restoring old behavior).

Reply:
Sounds good. It would be great in subsequent issue to find a way of handling opening the WebUI when using embedded nodes. Has this been looked at before?